java/buffer/README.md

```diff
@@ -1,25 +1,39 @@
 
-Creating buffers is a core concept in GIS proximity analysis, allowing you to visualize and locate geographic features contained within a polygon. For example, suppose you wanted to visualize areas of your city where alcohol sales are prohibited because they are within 500 meters of a school. The first step in this proximity analysis would be to generate 500 meter buffer polygons around all schools in the city. Any such businesses you find inside one of the resulting polygons are violating the law. If you are using planar buffers, make sure that the input locations and distance are suited to the spatial reference you're using. Remember that you can also create your buffers using geodesic and then project them to the spatial reference you need for display or analysis. For more information about using buffer analysis, see [How buffer analysis works](https://pro.arcgis.com/en/pro-app/tool-reference/analysis/how-buffer-analysis-works.htm) in the ArcGIS Pro documentation.  
+Create a buffer around a map point and display the results as a `Graphic`.
 
-![Buffer App](buffer.png)
+![Image of Buffer](buffer.png)
+
+## Use case
+
+Creating buffers is a core concept in GIS proximity analysis that allows you to visualize and locate geographic features contained within a polygon. For example, suppose you wanted to visualize areas of your city where alcohol sales are prohibited because they are within 500 meters of a school. The first step in this proximity analysis would be to generate 500 meter buffer polygons around all schools in the city. Any such businesses you find inside one of the resulting polygons are violating the law.
 
 ## How to use the sample
-1. Tap on the map.    
-2. A planar and a geodesic buffer will be created at the tap location using the distance (miles) specified in the text box.    
-3. Continue tapping to create additional buffers. Notice that buffers closer to the equator appear similar in size. As you move north or south from the equator, however, the geodesic polygons become much larger. Geodesic polygons are in fact a better representation of the true shape and size of the buffer.   
-4. Click `Clear` to remove all buffers and start again.
+
+1. Tap on the map.
+2. A planar and a geodesic buffer will be created at the tap location using the distance (miles) specified in the text box.
+3. Continue tapping to create additional buffers. Notice that buffers closer to the equator appear similar in size. As you move north or south from the equator, however, the geodesic polygons become much larger. Geodesic polygons are in fact a better representation of the true shape and size of the buffer.
+4. Tap `Clear` to remove all buffers and start again.
```

Comment:
How to use section should not have numbered steps. Also, should the explanation in section 3 be removed/moved to use case?? @TADraeseke @Rachael-E 

```suggestion
Tap on the map. A planar and a geodesic buffer will be created at the tap location using the distance (miles) specified in the text box. Continue tapping to create additional buffers. Notice that buffers closer to the equator appear similar in size. As you move north or south from the equator, however, the geodesic polygons become much larger. Geodesic polygons are in fact a better representation of the true shape and size of the buffer. Tap `Clear` to remove all buffers and start again.
```